smcarrera memory V2

smcarrera opened the pull request.

Commit:
memory V2

Commit:
memory v2 - with changes

added document.ready to js

Commit:
memory v2 - with changes

- added document. ready so JS would load
- added additional line at end of JS to pass test

Commit:
memory V2

I seem to be getting errors from loading the query.js file.

I was trying to instead use google’s hosted library, but the V2 no
longer works.

<script
src="//ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></scrip
t>

Any advice?

Commit:
memory V2 - latest changes

fixed using google’s library
should now also pass tests

Commit:
smcarrera memory V3 - WIP

hoping to complete tomorrow - adding now in case you’re grading

changes include:
*ability for the user to enter number of rows & columns to dynamically
create cards with basic form validation checks
- TODO additional validation checks, maybe limits on cols & rows
- Creation of a Card Object
- Dynamically adding IDs to the new divs
- TODO integration of memory logic from V2
- TODO shuffling of cards
- TODO CSS

Commit:
V3 CSS - WIP

added an additional line at the end

afeld commented:
Looks like the V3 is still in progress... you're on your way, so just let me know when it's ready for me to review. Take a look at [the build warnings](https://travis-ci.org/advanced-js/memory/builds/39119998#L73-L92), too. Thanks!

Commit:
smcarrera -oop memory - V4

had to start over since I got stuck with my initial attempt at V3 when
creating a dynamic # of cards.

This version (my V4) contains the following:
*now object oriented
* counts your turns and score
* has a rest button

Commit:
memory - removing the old files

Commit:
memory V4 - changes to pass tests

includes and additional line at the end of the css file
and changes in declarations of assignments in JS

Commit:
smcarrera memory V5

I took V4 and added the functionality again to generate your own memory
size

Commit:
memory V5 - changes to pass tests

changes include:
removed extra semi colons
changed the comparison operator from ‘==‘ to’===‘

also added some comments to V4 for clarification

Commit:
memory v5 - changes to pass test

removed reuse of variable name by removing unnecessary function

smcarrera commented:
went too far down the rabbit hole working with the dynamically generated memory cards and decided to re-start from scratch (which was much easier/faster in the end), so V3 files are deleted (although I reused a lot of logic)

V4 is my new V3 not including dynamically generated memory cards
V5 is the same thing, but with dynamically created memory cards where the user enters # rows & cols

CSS is still a to do but will save that for an iteration later on
I think in future iterations (post class :) I'd probably want to try this using a framework like bootstrap and making this work with images so it's not reliant on text color

Commit:
memory V5 - reset button changes

realized I didn’t clear the form inputs on reset I was only clearing
the memory cards, just added
